nucypher/blockchain/eth/actors.py

```diff
@@ -1042,6 +1063,25 @@ def prolong_stake(self,
         self.refresh_stakes()
         return receipt
 
+    @only_me
+    def merge_stakes(self,
+                     stake_1: Stake,
+                     stake_2: Stake
```

Comment:
These stakes are truly equal, or can one of them be considered the "base"?

Reply:
truly equal, order doesn't matter